index.html

```diff
@@ -390,6 +273,11 @@ <h3 class="form-title">Contact me</h3>
           </ul>
         </form>
       </section>
+      <section
+        class="modal-container"
+        id="modal-container"
+        onclick="closeModal()"
+      ></section>
     </main>
     <script src="script.js"></script>
```

Comment:
- [ ] Optional

- It will be nice to move the javascript `script tag` to the head section at the top and then add the `defer` attribute like so:
```
<head>
other scripts...
<script src="script.js" defer></script>
<title>----</title>
</head>  
```
This will allow your app load faster.... you can try it.

Reply:
Thanks for the great advice this was very helpful to me.

Reply:
@enaburekhan Would you please re-review my project? I did what you told me.